meson.build

```diff
@@ -147,19 +147,15 @@ if get_option('default_library') == 'shared'
   endif
 endif
 
-executable_cflags = []
-executable_linkflags = []
-library_cflags = ['-DRZ_PLUGIN_INCORE=1']
```

Comment:
This was the reason why there are separate cflags/ldflags i think ... but the pr lgtm and the ci is green